cli/up/up.go

```diff
@@ -98,20 +98,26 @@ func New() *cobra.Command {
 		"Leave containers running after blimp up exits")
 	cobraCmd.Flags().BoolVarP(&cmd.forceBuildkit, "remote-build", "", false,
 		"Force Docker images to be built in your sandbox instead of locally")
+
+	cobraCmd.Flags().BoolVarP(&cmd.disableStatusOutput, "disable-status-output", "", false,
+		"Don't print status updates. Used by preview implementation.")
```

Comment:
We should probably also disable the spinner on "Booting sandbox". It's fine to still print the message, but without backspace terminal codes it looks really weird.

Reply:
I didn't think it looked that bad. A little ugly, but not that noticeable.

Reply:
Okay, we can leave it for now.